src/components/ProductNameAndEdition.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from "react";
+
+type Content = {
+  colours: string[];
+  image: string[];
+  index: number;
+  name: string;
+  edition?: string;
+  price: string;
+};
```

Comment:
This seems to be used in quite a few places, why is it redeclared? If it's the same type, export and use the same type

Reply:
exported.